src/Features/Core/Portable/AddPackage/AbstractAddPackageCodeFixProvider.cs

```diff
@@ -46,31 +33,35 @@ internal abstract partial class AbstractAddPackageCodeFixProvider : CodeFixProvi
 
         var workspaceServices = document.Project.Solution.Services;
 
-        var symbolSearchService = _symbolSearchService ?? workspaceServices.GetService<ISymbolSearchService>();
-        var installerService = _packageInstallerService ?? workspaceServices.GetService<IPackageInstallerService>();
+        if (workspaceServices.GetService<ISymbolSearchService>() is not { } symbolSearchService ||
```

Comment:
fwiw, i don't like `not { } name` to mix the retrival and null check.  i'd prefer a distinct null check step.